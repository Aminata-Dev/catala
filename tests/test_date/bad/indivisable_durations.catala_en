## `Runtime.IndivisableDurations` exception handling

### Day divided by months

```catala
declaration scope DayByMonth:
  context d content decimal

scope DayByMonth:
  definition d equals 1 day /^ 2 month
```

```catala-test {id="DayByMonth.Interpret"}
catala Interpret -s DayByMonth
```
