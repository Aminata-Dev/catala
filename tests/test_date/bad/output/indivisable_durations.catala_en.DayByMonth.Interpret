[ERROR] Cannot divide durations that cannot be converted to a precise number of days

  --> tests/test_date/bad/indivisable_durations.catala_en
   | 
10 |   definition d equals 1 day /^ 2 month
   |                       ^^^^^
   + `Runtime.IndivisableDurations` exception handling
   +-+ Day divided by months

  --> tests/test_date/bad/indivisable_durations.catala_en
   | 
10 |   definition d equals 1 day /^ 2 month
   |                                ^^^^^^^
   + `Runtime.IndivisableDurations` exception handling
   +-+ Day divided by months
