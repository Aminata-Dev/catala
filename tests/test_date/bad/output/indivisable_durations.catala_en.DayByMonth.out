[ERROR] Cannot divide durations that cannot be converted to a precise number of days

  --> test_date/bad/indivisable_durations.catala_en
   | 
10 |   definition d equals 1 day /^ 2 month
   |                       ^^^^^
   + `Runtime.IndivisableDurations` exception handling
   +-+ Day divided by months

  --> test_date/bad/indivisable_durations.catala_en
   | 
10 |   definition d equals 1 day /^ 2 month
   |                                ^^^^^^^
   + `Runtime.IndivisableDurations` exception handling
   +-+ Day divided by months
